dev-haskell/vector/vector-0.12.1.2.ebuild

```diff
@@ -18,24 +18,29 @@ SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 IUSE="+boundschecks internalchecks unsafechecks"
 
-RESTRICT="test" # test-suite is too memory intensive
```

Comment:
Was it fixed since?

Reply:
I was able to run the tests without any noticeable RAM issues, i did monitor RAM usage during the test.

I wasn't sure if there was some upstream bug associated with this, but as the test ran fine I thought it'd be ok to unmask

Reply:
Ah, nice! I vaguely remember that with `HCFLAGS=-O0` tests were very hard to run successfully. If it's still the case we could filter out explicit `-O0`. If not we can just allow tests to run.

Reply:
I re-ran with `HCFLAGS=-O0 ebuild vector-0.12.1.2.ebuild test`. Assuming this is the appropriate way to duplicate the old issue, I can confirm that it ran fine. Seemed to use 71 MB or so of RAM.

Reply:
Yes, it should be fine assuming there is no existing unpacked directory. `HCFLAGS=-O0 ebuild ....ebuild clean test` should do it for sure.

Reply:
perfect, the directory was empty so we should be good.